### ISSUE TITLE
Confusing about contentDirInclude

### ISSUE THREAD

**Issue by teobler** [contentlayerdev/contentlayer#383]
Thanks for whole teams effort to build this amazing tool!

I'm struggling with contentDirInclude option here.

I have a folder structure looks like:
```
# NextJS root dir here
|— app
|— posts
|  |   |— some-post.md
|— public
|  |   |— some-special-page-need-to-be-processed.md
|— pages
|— components
```

And my config file looks like:
```
export const Post = defineDocumentType(() => ({
  name: "Post",
  filePathPattern: `posts/*.md`,
  fields: {},
}));

export const Other = defineDocumentType(() => ({
  name: "Other",
  filePathPattern: `public/*.md`,
  fields: {},
}));

export default makeSource({
  contentDirPath: ".",
  contentDirInclude: ["posts", "public"]
  documentTypes: [Post, Other],
  markdown: {
    remarkPlugins: [somePluginsHere],
    rehypePlugins: [somePluginsHere]
  },
});
```
I assume contentlayer will generate all md files to `posts` and `other` from `posts` and `public` folders for me.

But somehow this config hide all the md fies for me, it cannot scan any files, then I use `contentDirExclude` exclude all others folders for `posts` and `public`, it works, but I don't think this is the right way to use it, there will be a long list to exclude everything for just two folders.

And this issue may related to #254

**Comment by schickling** [comment]
Thanks for opening this issue. Can you please provide **minimal** a reproduction repo with instructions how to reproduce the problem you're experiencing? Thank you!

**Comment by schickling** [comment]
This should be addressed with the [`0.3.2` release](https://github.com/contentlayerdev/contentlayer/releases/tag/v0.3.2). 🎉
